### ISSUE TITLE
Add support for usage of typescript compiler 'paths' property

### ISSUE THREAD

**Issue by JoepKockelkorn** [retarsis/routerkit#47]
### What is this feature?

In setups where the typescript compiler 'paths' property is used (i.e. when using [nx](https://nx.dev/angular)), the parse schematic can't resolve the paths to lazy loaded modules.

### How the feature should work?

It can retrieve the relative path(s) from the typescript compiler config.

### Is your feature request related to a problem?

The 'error' of running `ng g @routerkit/core:parse --project app` is as following:
```
Can't find file with relative path @my-solution/app/feature/feature.module.ts
```

**Comment by retarsis** [comment]
Thanks for creating issue

Interesting 🤔
We recently implemented "ts paths" support for regular angular/cli application.

Is there a publicly available version or steps to reproduce the issue?

**Comment by JoepKockelkorn** [comment]
@retarsis No, but I can try to create one later today. I have a feeling it is the different naming of the tsconfig. In a nx repo `tsconfig.base.json` is used instead of `tsconfig.json`, but I have to verify this.

**Comment by retarsis** [comment]
@JoepKockelkorn that'd be awesome 😊

**Comment by JoepKockelkorn** [comment]
@retarsis I was able to reproduce it in [a minimal nx repo here](https://github.com/JoepKockelkorn/routerkit-paths-bug). Run `yarn install` and then `nx g @routerkit/core:parse --project app` to reproduce.

**Comment by retarsis** [comment]
@JoepKockelkorn many thanks 😌

we will take a look on it today with @limitofzero

**Comment by retarsis** [comment]
oh, merged PR autoclosed this issue
I'll release this fix in a few minutes

**Comment by retarsis** [comment]
so, this should now be fixed since version 0.7.2 😌

**Comment by retarsis** [comment]
so, this should now be fixed since version 0.7.2 😌

**Comment by JoepKockelkorn** [comment]
Yes, I can confirm [that it works after upgrading](https://github.com/JoepKockelkorn/routerkit-paths-bug/commit/c78d026d6d5a15aa6966999232bc2e4b6b6d2823).
